trl/trainer/online_dpo_trainer.py

```diff
@@ -63,12 +63,11 @@ def __init__(
         self.args = config
         args = config
         self.tokenizer = tokenizer
-        self.policy = policy
 
-        self.policy.generation_config.eos_token_id = (
-            None  # disable `pad_token_id` and `eos_token_id` because we just want to
-        )
-        self.policy.generation_config.pad_token_id = None  # generate tokens without truncation / padding
+        # disable `pad_token_id` and `eos_token_id` because we just want to
+        policy.generation_config.eos_token_id = None
+        # generate tokens without truncation / padding
+        policy.generation_config.pad_token_id = None
 
         self.ref_policy = ref_policy
```

Comment:
can we change this to `ref_model` in this refactor? Other leave it to another PR.